src/KPI.ts

```diff
@@ -0,0 +1,43 @@
+// (C) 2020 GoodData Corporation
+
+import { IObjectMeta } from './Meta';
+import { ExtendedDateFilters } from './ExtendedDateFilters';
+
+export interface IKPI {
+    meta: IObjectMeta;
+    content: IKpiContentWithoutComparison | IKpiContentWithComparison;
+}
+
+export interface IKpiContentWithComparison {
```

Comment:
Suggestion: you could probably extract properties that are in both `IKpiContentWithComparison` and `IKpiContentWithoutComparison` into something like `IKpiContentBase` to reduce duplication. You can then write 

```ts
export interface IKpiContentWithComparison extends IKpiContentBase {
  comparisonType: IKpiComparisonTypeComparison;
  comparisonDirection: IKpiComparisonDirection;
}
```